restful-ws/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "Jakarta RESTful Web Services Specification 2.1"
+summary: "First release for Jakarta EE 8"
```

Comment:
The title should not include the version number and the summary should be a copy of the scope statement.